.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   - pull_request
 
 env:
-  GO_VERSION: "1.19.x"
+  GO_VERSION: "1.20.x"
```

Comment:
we can probably get away with setting this to `"stable"` and be done with constantly needing to update it
https://github.com/actions/setup-go#using-stableoldstable-aliases

Reply:
Hmm, the only thing is it might be annoying if we're randomly getting linter errors like we did here or random breaks like we've had in the past when some packages have breaking changes across golang versions.